src/core/Timepicker/BaseTimePicker.tsx

```diff
@@ -0,0 +1,26 @@
+import 'date-fns';
+import React, { useState } from 'react';
+import MomentUtils from '@date-io/moment';
+import { MuiPickersUtilsProvider, TimePicker } from '@material-ui/pickers';
+import { BaseTimePickerProps } from './base';
+
+interface TimePickerProps extends BaseTimePickerProps {
+  value?: Date;
+}
+
+const BaseTimePicker: React.FC<TimePickerProps> = ({ value }) => {
+  const date = value != undefined ? value : new Date();
+  const [selectedDate, handleDateChange] = useState<Date | null>(date);
```

Comment:
Date isn't required in time picker

Reply:
@S-ayanide that is , the value shouldn't be passes?

Reply:
No, Date shouldn't be passed